### DISCUSSION TITLE
How to use JS event when lazy on?

### DISCUSSION THREAD

**Discussion by SmoKE585** [livewire/livewire#6700] [1. Help] [answered]
Iam use lazy load nasted component and event **livewire:initialized** not runing in my code.
Nested component:
`<script>
        document.addEventListener('livewire:initialized', () => {
            alert(123);
        })
    </script>`
Component:
`<livewire:components.goods.product-images good_id="{{$product->good_id}}" lazy="on-load"/>`

What am I doing wrong?

- Laravel 10
- Livewire 3

**Comment by N1ebieski** [comment, marked as answer]
Why not a Alpine?

```html
<div x-data x-init="alert(123)"></div>
```

**Reply by SmoKE585** [reply]
I forgot about him. 😅
